apps/explorer/src/modules/txs/components/TableTxs.vue

```diff
@@ -381,7 +363,7 @@ export default class TableTxs extends TableTxsMixin {
       ===================================================================================
       */
 
-  get loading() {
+  get loading(): boolean | undefined {
```

Comment:
Same here, does  return undefined?

Reply:
since it return this.$apollo.loading || this.syncing , syncing types are boolean | undefined